.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "omnisharp.organizeImportsOnFormat": true
+    "omnisharp.organizeImportsOnFormat": true,
+    "dotnet.defaultSolution": "msgraph-cli-beta.sln"
```

Comment:
wouldn't it make sense to also update the solution name to align everything and avoid confusion?

Reply:
it's just from a consistency perspective

Reply:
The solution file name matching the binary archive name isn't a convention we have. If we added that as a rule, it means that any time we change that again, we have to rename files again.

Reply:
I believe it will add an unnecessary maintenance burden.

Reply:
since this is the only solution on the repo, we're not passing the solution name to any of the dotnet command calls.
The only reference to that name is here https://github.com/microsoftgraph/msgraph-beta-cli/blob/75e88b30670d3b7336a6c301bde6416e817e2067/.azure-pipelines/release-cli.yaml#L13

[a simple search](https://github.com/search?q=repo%3Amicrosoftgraph%2Fmsgraph-beta-cli%20msgraph-cli-beta&type=code) would suggest you've already changed most references. What'd be left:
- sln file name
- csproj file name
- csproj references in tasks.json
- csproj references in sln file (no, the guids don't need to be changed)

Reply:
I'd like to add this to the NuGet work I'm meant to be doing. That way I update the pipeline once.

Reply:
https://github.com/microsoftgraph/msgraph-beta-cli/issues/10

Reply:
I've made the changes to move this along quicker.